sgpt/config.py

```diff
@@ -43,6 +47,7 @@ def init() -> None:
         config["CACHE_LENGTH"] = os.getenv("CACHE_LENGTH", str(CACHE_LENGTH))
         config["CACHE_PATH"] = os.getenv("CACHE_PATH", str(CACHE_PATH))
         config["REQUEST_TIMEOUT"] = os.getenv("REQUEST_TIMEOUT", str(REQUEST_TIMEOUT))
+        config["DEFAULT_MODEL"] = os.getenv("DEFAULT_MODEL", str(DEFAULT_MODEL))
```

Comment:
This will break `sgpt` for everyone that will upgrade to the new version. All current `sgpt` users already have a config file. This will not update it, it will create a new config file only if it is not exists. I will fix it in my branch after merge.